telebot/async_telebot.py

```diff
@@ -511,7 +511,6 @@ async def _run_middlewares_and_handlers(self, message, handlers, middlewares, up
                     if not process_update: continue
                     for i in signature(handler['function']).parameters:
                         params.append(i)
-                    result = None
```

Comment:
Again, any reason?
And, one topic - one PR

Reply:
> Again, any reason?
Just no need in this line, because result is assinged in every branch.

> And, one topic - one PR
No functional updates. It is just IDE hints solving.